spec/models/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  fixtures :users
+  context 'create a simple user when have a valid input' do
+    it 'check if the input is valid' do
+      # Setup & Exercise
+      user = User.new(name: 'usuario')
+      # Verify
+      expect(user.name).to match(/[a-zA-Z]/)
```

Comment:
This expectation is checking against a regular expression unnecessarily and that could mask a range of implementation errors. The goal of writing specs is ensuring the the implementation behave as expected, but suppose someone updates the model class in a way that it stores a string other than the name provided to the database, your test would still pass.

Reply:
You could write this test as simple as:
```rb
expect(user).to be_valid
```